config/clusters/jupyter-health/common.values.yaml

```diff
@@ -0,0 +1,181 @@
+nfs:
+  enabled: true
+  pv:
+    enabled: true
+    # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+    mountOptions:
+      - rsize=1048576
+      - wsize=1048576
+      - timeo=600
+      - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+      - retrans=2
+      - noresvport
+    serverIP: fs-07b67adf6626672af.efs.us-east-2.amazonaws.com
+    baseShareName: /
+jupyterhub:
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    jupyterhubConfigurator:
+      enabled: false
+    homepage:
+      templateVars:
+        org:
+          name: Jupyter Health
+          logo_url: https://jupyter.org/assets/homepage/main-logo.svg
+          url: https://github.com/jupyterhealth/
+        designed_by:
+          name: "Jupyter Health"
+          url: https://github.com/jupyterhealth/
+        operated_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        funded_by:
+          name: "Jupyter Health"
+          url: https://www.earthdata.nasa.gov/esds/veda
+  hub:
+    allowNamedServers: true
+    config:
+      JupyterHub:
+        authenticator_class: github
+      GitHubOAuthenticator:
+        allowed_organizations:
+          - jupyterhealth
+        scope:
+          - read:org
+      Authenticator:
+        admin_users:
+          - minrk
+          - fperez
+          - colliand
+          - maryamv
+          - ryanlovett
+  singleuser:
+    defaultUrl: /lab
+    profileList:
+      - display_name: "Pangeo Notebook"
+        slug: pangeo
+        description: Pangeo based notebook with a Python environment
+        default: true
+        kubespawner_override:
+          image: quay.io/pangeo/pangeo-notebook:2024.03.22
+          init_containers:
+            # Need to explicitly fix ownership here, as otherwise these directories will be owned
+            # by root on most NFS filesystems - neither EFS nor Google Filestore support anonuid
+            - name: volume-mount-ownership-fix
+              image: busybox:1.36.1
+              command:
+                - sh
+                - -c
+                - id && chown 1000:1000 /home/jovyan /home/jovyan/shared && ls -lhd /home/jovyan
+              securityContext:
+                runAsUser: 0
+              volumeMounts:
+                - name: home
+                  mountPath: /home/jovyan
+                  subPath: "{username}"
+                # mounted without readonly attribute here,
+                # so we can chown it appropriately
+                - name: home
+                  mountPath: /home/jovyan/shared
+                  subPath: _shared
```

Comment:
We are already providing an init container by default to do this - aren't we?

Reply:
Yep! Another reason to move away from copy pasting hubs! This was leftover from copy pasting from nasa-veda